vnext/Desktop/Modules/WebSocketModule.cpp

```diff
@@ -204,7 +213,29 @@ shared_ptr<IWebSocketResource> WebSocketModule::GetOrCreateWebSocket(int64_t id,
       if (!strongInstance)
         return;
 
-      auto args = dynamic::object("id", id)("data", message)("type", isBinary ? "binary" : "text");
+      dynamic args = dynamic::object("id", id)("data", message)("type", isBinary ? "binary" : "text");
+
+      auto contentHandler = IWebSocketModuleContentHandler::GetInstance().lock();
+      if (isBinary)
+      {
+        auto buffer = CryptographicBuffer::DecodeFromBase64String(Utf8ToUtf16(message));
+        winrt::com_array<uint8_t> arr;
+        CryptographicBuffer::CopyToByteArray(buffer, arr);
+        auto data = std::vector<uint8_t>(arr.begin(), arr.end());
```

Comment:
Should this be a streaming api in the future? I know we don't have ProcessMessage, but now we allocate and copy the message 3 times..

Reply:
Likely yes.

We may move out of `std::vector<uint8_t>` to simplify this process.